bundles/org.openhab.binding.folderwatcher/pom.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.openhab.addons.bundles</groupId>
+    <artifactId>org.openhab.addons.reactor.bundles</artifactId>
+    <version>2.5.4-SNAPSHOT</version>
```

Comment:
```suggestion
    <version>2.5.5-SNAPSHOT</version>
```

Reply:
This generates build error: Project build error: Non-resolvable parent POM for org.openhab.addons.bundles:org.openhab.binding.folderwatcher:2.5.5-SNAPSHOT: Could not find 
 artifact org.openhab.addons.bundles:org.openhab.addons.reactor.bundles:pom:2.5.5-SNAPSHOT and 'parent.relativePath' points at wrong local POM

Reply:
Well, you might need to pull the latest 2.5.x branch first. But you definitely need to change to `2.5.5-SNAPSHOT` before this PR can be merged. I assure you that it is a required change.